packages/storefront-events-collector/README.md

```diff
@@ -100,10 +100,9 @@ The handlers forward events to two edges:
     B2B events
 
     `createRequisitionList`
-- 
-    `deleteRequisitionList`
-- 
-    `addToRequisitionList`
+
+-   `deleteRequisitionList`
+-   `addToRequisitionList`
```

Comment:
Not sure why these are formatted as a list while the rest of the section isn't? Maybe just make them consistent with the other events in this section.

Reply:
This change was enforced by the linter/formatter.  I will look into this, but I think it will fail deploy if changedd.